orchestrator/nodes/template/template.html

```diff
@@ -73,7 +73,7 @@ <h3>Details</h3>
             field: {value:"payload", validate: RED.validators.typedInput("fieldType")},
             fieldType: {value:"msg"},
             syntax: {value:"handlebars"},
-            template: {value:RED._('dojot/template:template_value') + " {{{stringify payload}}} !"},
+            template: {value:RED._('dojot/template:template_value') + "{{payload}} !"},
```

Comment:
Why this was removed?

Reply:
Because it does not belong to this PR, I accidentally add it to execute some tests.
These changes are available on feature/template-node-stringify.